app/views/rent_mailer/new_rent_email.slim

```diff
@@ -0,0 +1,12 @@
+
+h1 
+  | You have registered a new rent for the book 
+  = @rent.book.title
+p 
+  | Starting on: 
+  b
```

Comment:
If I can do one comment, we prefer using CSS for all UI. But in this case let it as this